cfile/hogfile.h

```diff
@@ -62,9 +62,11 @@
 
 #include <cstdint>
 #include <cstdio>
+#include <string>
 
 #define HOG_HDR_SIZE (64)
 #define HOG_TAG_STR "HOG2"
+#define HOG_TAG_LEN std::string::traits_type::length(HOG_TAG_STR)
```

Comment:
No need to call std::string::length, we already knows that there always be 4 bytes. Just define it as is:

```cpp
#define HOG_TAG_LEN (4)
```

Reply:
OK, done.